Kernel/Arch/riscv64/InterruptManagement.cpp

```diff
@@ -37,9 +37,9 @@ void InterruptManagement::find_controllers()
     TODO_RISCV64();
 }
 
-u8 InterruptManagement::acquire_mapped_interrupt_number(u8)
+u8 InterruptManagement::acquire_mapped_interrupt_number(u8 original_irq)
```

Comment:
Perhaps a future FIXME is in order for this, to split out the "mapped interrupt" concept to be x86-only? iirc this is for IOAPIC stuff right?

Reply:
Hm, I think this is fine for now as long as we assume that an irq number is unique for all interrupt controllers, as certain architectures might have different local interrupt numbers.
But we should remove this once we refactor interrupt handling.